js/core/utils.js

```diff
@@ -27,15 +27,48 @@ define(
             return $(arr);
         };
 
-        // For any element, returns a title string that applies the algorithm used for determining the
+        // For any element, returns an array of title strings that applies
+        // the algorithm used for determining the
         // actual title of a <dfn> element (but can apply to other as well).
-        $.fn.dfnTitle = function () {
-            var title;
-            if (this.attr("title")) title = this.attr("title");
+        // 
+        // if isDefinition is true, then the element is a definition, not a 
+        // reference to a definition.  Any @title or @lt will be replaced with
+        // @data-lt to be consistent with Bikeshed / Shepherd.
+        $.fn.dfnTitle = function ( isDefinition ) {
+            var titles = [];
+            var theAttr = "";
+            var titleString = ""; 
+            if (this.attr("title")) {
+                titleString = this.attr("title");
+                theAttr = "title";
+            }
+            else if (this.attr("lt")) {
+                titleString = this.attr("lt");
+                theAttr = "lt";
+            }
             else if (this.contents().length == 1 && this.children("abbr, acronym").length == 1 &&
-                     this.find(":first-child").attr("title")) title = this.find(":first-child").attr("title");
-            else title = this.text();
-            return title.toLowerCase().replace(/^\s+/, "").replace(/\s+$/, "").split(/\s+/).join(" ");
+                     this.find(":first-child").attr("title")) titleString = this.find(":first-child").attr("title");
+            else titleString = this.text();
+            // now we have a string of one or more titles
+            // ensure there is no extra whitespace
+            titleString = titleString.toLowerCase().replace(/^\s+/, "").replace(/\s+$/, "").split(/\s+/).join(" ");
```

Comment:
can you put each `.xxx` on a separate line? Makes it easier to read. It also helpful to add comments for each regex or an explanation at the top:
```JS
// Explanation of the following... 
titleString = titleString.toLowerCase()
                     .replace(/^\s+/, "") // or single line explanation... 
                     .replace(/\s+$/, "")
                     .split(/\s+/)
                     .join(" ");
```